### DISCUSSION TITLE
Modify Existing Container?

### DISCUSSION THREAD

**Discussion by richip** [itzg/docker-minecraft-server#752] [Q&A] [answered]
Is it possible to modify the configurations for an existing docker container? Things like modifying memory options, JVM_OPTS, adding plugins, etc. I suppose I could rephrase the question to could I use docker commands to manage the documented creation/run parameters of the docker-minecraft-server?

**Comment by itzg** [comment, marked as answer]
That's a very general Docker question, but basically, no, you can't. [From the commands list](https://docs.docker.com/engine/reference/commandline/docker/) an "edit" command is unfortunately missing. You'll need to instead remove and recreate the container.

I recommend using docker compose to avoid that annoyance since compose will seamless take care of container recreation for you when editing the environment variables, etc.
